.github/workflows/release.yaml

```
@@ -0,0 +1,41 @@
+---
+name: Gem Release
+
+on:
+  push:
+    tags:
+      - '*'
```

Should we add the filter, just in case?

```suggestion
      - 7.**
```

---

:+1: